client/lib/analytics/index.js

```diff
@@ -282,12 +282,17 @@ const analytics = {
 			analytics.emit( 'record-event', eventName, eventProperties );
 		},
 
-		recordPageView: function( urlPath ) {
+		recordPageView: function( urlPath, params ) {
 			let eventProperties = {
 				path: urlPath,
 				do_not_track: doNotTrack() ? 1 : 0
 			};
 
+			// add optional path params
+			if ( params ) {
+				eventProperties = assign( eventProperties, params );
```

Comment:
`lodash.assign` mutates the `eventProperties` object. Assigning the return value to `eventProperties` is not necessary. But also not harmful :)

Reply:
thx. I tend to keep redundant assignment for readability sake.